src/ome.ts

```diff
@@ -0,0 +1,226 @@
+        let l = createLoader([data]);
+        return l;
+    } catch (err) {
+        console.log(`Missing Well at ${path}`);
+    }
+}
+
+export async function loadOMEWell(config: ImageLayerConfig, store: HTTPStore, rootAttrs: RootAttrs): Promise<SourceData> {
+    const { acquisition } = config;
+    let acquisitions = [];
+
+    const wellAttrs = rootAttrs.well as OmeWellData;
+    if (!wellAttrs.images) {
+        throw Error(`Well .zattrs missing images`);
+    }
+    const [row, col] = store.url.split('/').filter(Boolean).slice(-2);
+
+    let images = wellAttrs.images;
+
+    // Do we have more than 1 Acquisition?
+    const acqIds = images.flatMap(img => img.acquisition ? [img.acquisition] : []);
+    if (acqIds.length > 1) {
+        // Need to get acquisitions metadata from parent Plate
+        const plateUrl = store.url.replace(`/${row}/${col}`, '');
+        const plateStore = new HTTPStore(plateUrl);
+        const plateAttrs = await getJson(plateStore, `.zattrs`);
+        acquisitions = plateAttrs?.plate?.acquisitions;
+
+        // filter imagePaths by acquisition
+        if (acquisition) {
+            images = images.filter(img => img.acquisition == parseInt(acquisition));
+        }
+    }
+
+    let imagePaths = images.map(img => img.path + (img.path.endsWith('/') ? '' : '/'));
+
+    // Use first image for rendering settings, resolutions etc.
+    let imageAttrs = await getJson(store, `${imagePaths[0]}.zattrs`)
+    // Lowest resolution is the 'path' of the first 'dataset' from the first multiscales
+    let resolution = imageAttrs.multiscales[0].datasets[0].path;
+
+    // Create loader for every Image.
+    const promises = imagePaths.map(p => createLoaderFromPath(store, `${p}${resolution}/`));
+    let loaders = await Promise.all(promises);
+    const loader = loaders.find(Boolean);
+
+    // Load Image to use for channel names, rendering settings, sizeZ, sizeT etc.
+    const sourceData: SourceData = loadOME(config, imageAttrs.omero, loader as ZarrLoader);
+    const cols = Math.ceil(Math.sqrt(imagePaths.length));
+
+    if (acquisitions.length > 0) {
+        // To show acquisition chooser in UI
+        sourceData.acquisitions = acquisitions;
+        sourceData.acquisition = acquisition || '-1';
+    }
+
+    sourceData.loaders = loaders;
+    sourceData.name = `Well ${row}${col}`;
+    sourceData.rows = Math.ceil(imagePaths.length / cols);
+    sourceData.columns = cols;
+    sourceData.onClick = (info: any) => {
+        let gridCoord = info.gridCoord;
+        if (!gridCoord) {
+            return;
+        }
+        const { row, column } = gridCoord;
+        let { source } = sourceData;
+        if (typeof source === 'string' && !isNaN(row) && !isNaN(column)) {
+            const field = (row * cols) + column;
+            if (source.endsWith('/')) {
+                source = source.slice(0, -1);
+            }
+            let imgSource = `${source}/${imagePaths[field]}`;
```

Comment:
probably another place to use a util like `ensureEndsWith`:

```suggestion
            source = ensureEndsWith(source, '/');
            let imgSource = `${source}${imagePaths[field]}`;
```

Reply:
I've created a util `rstrip(source, '/')` (same as python). I like seeing the `/` when creating paths like ```let imgSource = `${source}/${imagePaths[field]}`;``` since it helps to see the separate parts of the path.